packages/gitbook/src/components/PageAside/PageAside.tsx

```diff
@@ -62,10 +73,8 @@ export function PageAside(props: {
                 'py-8',
                 'break-anywhere', // To prevent long words in headings from breaking the layout
                 'lg:h-full',
-                withHeaderOffset ? 'lg:max-h-[calc(100vh_-_4rem)]' : 'lg:max-h-[100vh]',
-                withHeaderOffset ? 'top-16' : 'top-0',
                 'h-[100vh]',
-
+                'page-api-block:xl:max-2xl:z-10',
```

Comment:
I am not sure if this should be here?

Reply:
Without I was losing hover in some cases. Maybe could set it to `z-[1]` instead.